DZSwiftUI/DZSwiftUI/ProductView.swift

```diff
@@ -0,0 +1,206 @@
+                    productViewModel.isFiltersViewShown.toggle()
+                }, label: {
+                    Image(systemName: Constants.filterButtonImageName)
+                        .resizable()
+                        .frame(
+                            width: 30,
+                            height: 30
+                        )
+                        .foregroundColor(.white)
+                        .padding(.trailing, 30)
+                })
+            }
+        }
+    }
+
+    private var productsTotalPriceView: some View {
+        Text("\(Constants.totalPriceText) \(productViewModel.productsTotalPrice) \(Constants.cashSymbol)")
+            .onAppear {
+                productViewModel.countTotalSum()
+            }
+            .frame(
+                width: 250,
+                height: 50
+            )
+            .multilineTextAlignment(.center)
+            .font(.system(
+                size: 25,
+                weight: .bold
+            ))
+    }
+
+    private var productsPresentationView: some View {
+        VStack {
+            ForEach(productViewModel.products, id: \.id) { product in
+                makeProductsView(product)
+            }
+        }
+    }
+
+    // MARK: - Private Methods
+
+    private func makeProductsView(_ product: Product) -> some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: 10)
+                .frame(
+                    width: 350,
+                    height: 110
+                )
+                .foregroundColor(.white)
+                .shadow(radius: 5)
+            HStack {
+                Image(systemName: product.productImage)
+                    .resizable()
+                    .scaledToFit()
+                    .frame(
+                        width: 70,
+                        height: 80
+                    )
+                    .foregroundColor(.yellow)
+                VStack {
+                    Text(product.title)
+                        .frame(width: 100)
+                        .multilineTextAlignment(.center)
+                        .font(.system(
+                            size: 23,
+                            weight: .bold
+                        ))
+                    HStack {
+                        Button(action: {
+                            productViewModel.changeProductCount(-1, currentProduct: product)
+                        }, label: {
+                            Image(systemName: Constants.minusButtonImageName)
+                                .foregroundColor(.black)
+                        })
+                        Text("\(product.amount)")
+                            .frame(
+                                width: 30,
+                                height: 30
+                            )
+                            .padding(.all, 5)
+                            .border(.gray)
+                            .font(.title2)
+                        Button(action: {
```

Comment:
```suggestion
                        Button {
```
поправь в остальных кнопках тоже